internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -498,6 +501,25 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{RequeueAfter: shortWait}, nil
 	}
 
+	watchNamespace := os.Getenv("WATCH_NAMESPACE")
+
+	cm := &unstructuredv1.Unstructured{}
+	cm.SetGroupVersionKind(schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"})
+
+	if err := r.client.Get(ctx, types.NamespacedName{Namespace: watchNamespace, Name: "cf-crossplane"}, cm); err != nil {
+		log.Debug(errGet, "error", err)
+		return reconcile.Result{}, errors.Wrap(resource.IgnoreNotFound(err), errGet)
+	}
+
+	data := cm.Object["data"].(map[string]interface{})
+
+	flag := data["gracfullyDelete"]
```

Comment:
Two things:
- typo in the name
- gracfullyDelete implies we will delate all crossplane, while this flag only removed finalizers. Perhaps the name shall be a bit more specific?